### ISSUE TITLE
'hiera' is deprecated in favor of using 'lookup'

### ISSUE THREAD

**Issue by mphilipps** [example42/puppi#154]
hi,
We keep getting the following warning in our puppetserver log:

```
WARN  [qtp2145888459-158] [puppetserver] Puppet The function 'hiera' is deprecated in favor of using 'lookup'. See https://puppet.com/docs/puppet/6.2/deprecated_language.html
   (file & line not available)
```
Even though puppet can't tell  me the file & line I think I have located the issue in `lib/puppet/functions/params_lookup.rb`.

**Comment by alvagante** [comment]
Yes hiera is deprecated, but for backward compatibility with Puppet 3 we have to keep on using it.

**Comment by mphilipps** [comment]
Last year Puppet 6 was released, do you really want to keep backward compatibility with Puppet 3?

**Comment by alvagante** [comment]
Yes, considering that we are talking about a warning and there're still Puppet3 installations around.
Anyway, since the issue is with the function using the new api, I will use the lookup function here, and keep hiera in the old api function version.
Keep tuned.
